### DISCUSSION TITLE
Error: The Service account needs the following permissions on the pod resource in the 'arc-runners' namespace.

### DISCUSSION THREAD

**Discussion by haleviyi** [actions/actions-runner-controller#3197] [Questions]
Hi,

I am using Kubernetes mode, but I am getting the following error:

Error: Error: The Service account needs the following permissions [{"group":"","verbs":["get","list","create","delete"],"resource":"pods","subresource":""},{"group":"","verbs":["get","create"],"resource":"pods","subresource":"exec"},{"group":"","verbs":["get","list","watch"],"resource":"pods","subresource":"log"},{"group":"batch","verbs":["get","list","create","delete"],"resource":"jobs","subresource":""},{"group":"","verbs":["create","delete","get","list"],"resource":"secrets","subresource":""}] on the pod resource in the 'arc-runners' namespace. Please contact your self hosted runner administrator.
Error: Process completed with exit code 1.
Error: Executing the custom container implementation failed. Please contact your self hosted runner administrator

 Any idea how to solve it? Do you know which Role I should update?
 
Thanks,
Yitzhak

**Comment by brett-au** [comment]
I think this is the same issue here - https://github.com/actions/actions-runner-controller/discussions/3160

**Comment by cjvirtucio87** [comment]
~I'm running into the same issue. I tried setting the `ACTIONS_RUNNER_CONTAINER_TEMPLATE` variable while mounting a `ConfigMap` according to [this answer](https://github.com/actions/actions-runner-controller/issues/2992#issuecomment-1764855221), but no luck.~

fixed a permission with the `Role`, and now it works. try setting the `serviceAccount` using a pod template in a `ConfigMap` according to the answer linked above.
